src/lib/Components/ArtworkGrids/InfiniteScrollGrid.tsx

```diff
@@ -49,37 +49,37 @@ interface ArtworksConnection {
 
 interface Props extends ArtistRelayProps, GeneRelayProps {
   /** The direction for the grid, currently only 'column' is supported . */
-  sectionDirection: string
+  sectionDirection?: string
 
   /** The arity of the number of sections (e.g. columns) to show */
-  sectionCount: number
+  sectionCount?: number
 
   /** The inset margin for the whole grid */
-  sectionMargin: number
+  sectionMargin?: number
 
   /** The per-item margin */
-  itemMargin: number
+  itemMargin?: number
 
   /** The artist in question */
-  artist: any
+  artist?: any
 
   /** The gene in question */
-  gene: any
+  gene?: any
```

Comment:
This component knowing of both artist and gene is a smell, it should be more generic.

Reply:
There’s now also the ‘sales’ grid, which will suffer from the same. /cc @ashfurrow

Reply:
I was thinking about this the other day, and it might be time to backport the backport back to Emission: https://github.com/artsy/force/blob/master/desktop/components/react/masonry_grid/MasonryGrid.js

When I was working on new Force responsive views I needed a grid that allowed me to pass in any kind of artwork and made a few modifications: https://github.com/artsy/force/blob/master/desktop/apps/auction/components/artwork_browser/main/ArtworkDisplay.js#L84

Reply:
Something like that yeah, let’s pair on that. The thought I had was that all sorts of types can have artwork connections and we could just have them implement an `ArtworkInterface` on their edge so that we can pass in any connection that implements that interface.